Create VisitSettle_Keyed.xml

Natural-Brother opened the pull request:
I didn't understood the line 52: **"Enable minifying penalty"**
and the line 56: **"goodwill"**, what is it?

I also forgot to change the first line **"Enter Settlement"** to **"Entrer dans la Colonie"**. 
i don't understand how to change back the file.

Natural-Brother committed:
Create VisitSettle_Keyed.xml

I didn't understood the line 52: "Enable minifying penalty"
and the line 56: "goodwill", what is it?

alt44s commented:
"Enable minifying penalty" is referring to the relations penalty given to the player with a certain faction when they uninstall any furniture on the other factions' settlement map. Meaning if a player uninstalls (minifies) any piece of furniture, their "goodwill" will drop.

"Goodwill" is the amount of relation points with a faction.

To change the file you need to make another commit to this PR and push any local changes you've made to it.

Natural-Brother committed:
Update VisitSettle_Keyed.xml

changed line 4: "Enter Settlement" to "Entrer dans la Colonie",
changed line 52: "minifying" to "désinstallation",
same for line 62

and other minor changes i don't remember.

alt44s commented:
Hi. Pushed an update to steam. What name do you want me to credit you with, and do you want to include any links?

Natural-Brother commented:
> Hi. Pushed an update to steam. What name do you want me to credit you with, and do you want to include any links?

i would like to be credited as 'petit slimy' please, 
thank you
